include/tdnftypes.h

```diff
@@ -311,9 +311,9 @@ typedef struct _TDNF_REPO_DATA
     char* pszUser;
     char* pszPass;
     int nPriority;
-    int nTimeout;
-    int nMinrate;
-    int nThrottle;
```

Comment:
In `repolist.c` we have: `pRepo->nTimeout = strtoi(cn->value);` - if we are so pedantic, shouldn't we change that to use `strtol()` too?